glide.lock

```diff
@@ -43,7 +43,7 @@ imports:
   - svc
   - winapi
 - name: github.com/davecgh/go-spew
-  version: 6cf5744a041a0022271cefed95ba843f6d87fd51
+  version: 5215b55f46b2b919f50a1df0eaa5886afe4e3b3d
```

Comment:
Right.  For some reason `glide update` is pulling the old one even after nuking the vendor directory.  I manually fixed it and pushed the update.